### ISSUE TITLE
Uncaught TypeError: Cannot read property 'getSelectedBufferRanges' of null

### ISSUE THREAD

**Issue by rugk** [atom/find-and-replace#659]
[Enter steps to reproduce below:]

0. Click on the markdown preview window and press Ctrl+F.
1. Happend every time I click on any modfier button (regex, only selection, ...) of the search and replace window

**Atom Version**: 1.4.2
**System**: Windows 7
**Thrown From**: [aligner](https://github.com/adrianlee44/atom-aligner) package, v0.18.1

### Stack Trace

Uncaught TypeError: Cannot read property 'getSelectedBufferRanges' of null

```
At C:\Users\%username%\AppData\Local\atom\app-1.4.2\resources\app.asar\node_modules\find-and-replace\lib\find-view.js:788

TypeError: Cannot read property 'getSelectedBufferRanges' of null
    at FindView.module.exports.FindView.anyMarkersAreSelected (C:\Users\%username%\AppData\Local\atom\app-1.4.2\resources\app.asar\node_modules\find-and-replace\lib\find-view.js:788:36)
    at FindView.anyMarkersAreSelected (C:\Users\%username%\AppData\Local\atom\app-1.4.2\resources\app.asar\node_modules\find-and-replace\lib\find-view.js:3:61)
    at FindView.module.exports.FindView.toggleRegexOption (C:\Users\%username%\AppData\Local\atom\app-1.4.2\resources\app.asar\node_modules\find-and-replace\lib\find-view.js:799:17)
    at HTMLButtonElement.<anonymous> (C:\Users\%username%\AppData\Local\atom\app-1.4.2\resources\app.asar\node_modules\find-and-replace\lib\find-view.js:3:61)
    at HTMLButtonElement.jQuery.event.dispatch (C:\Users\%username%\AppData\Local\atom\app-1.4.2\resources\app.asar\node_modules\atom-space-pen-views\node_modules\space-pen\node_modules\jquery\dist\jquery.js:4435:9)
    at HTMLButtonElement.elemData.handle (C:\Users\%username%\AppData\Local\atom\app-1.4.2\resources\app.asar\node_modules\atom-space-pen-views\node_modules\space-pen\node_modules\jquery\dist\jquery.js:4121:28)
```

### Commands

```
  4x -0:52.7.0 core:move-left (atom-text-editor.editor.mini.is-focused)
  6x -0:51.2.0 core:move-right (atom-text-editor.editor.mini.is-focused)
 14x -0:48.7.0 core:select-right (atom-text-editor.editor.mini.is-focused)
     -0:34.9.0 core:select-all (atom-text-editor.editor.mini.is-focused)
     -0:34.6.0 core:copy (atom-text-editor.editor.mini.is-focused)
     -0:32.7.0 core:select-all (atom-text-editor.editor.mini.is-focused)
     -0:26 core:backspace (atom-text-editor.editor.mini.is-focused)
     -0:24.5.0 core:move-left (atom-text-editor.editor.mini.is-focused)
     -0:23.1.0 core:select-left (atom-text-editor.editor.mini.is-focused)
     -0:22.7.0 core:copy (atom-text-editor.editor.mini.is-focused)
  4x -0:22.1.0 core:move-left (atom-text-editor.editor.mini.is-focused)
     -0:21.3.0 core:paste (atom-text-editor.editor.mini.is-focused)
  2x -0:20.9.0 core:move-left (atom-text-editor.editor.mini.is-focused)
     -0:20.4.0 core:paste (atom-text-editor.editor.mini.is-focused)
  4x -0:20.1.0 core:move-left (atom-text-editor.editor.mini.is-focused)
     -0:19.3.0 core:paste (atom-text-editor.editor.mini.is-focused)
```

### Config

```json
{
  "core": {
    "disabledPackages": [
      "autocomplete-emojis",
      "metrics",
      "tree-view-background",
      "markdown-table-formatter",
      "atom-beautify",
      "linter-js-standard",
      "linter-flake8",
      "tree-view-breadcrumb",
      "copy-paste",
      "linter-jsonlint",
      "line-diff-details",
      "atom-autocomplete-php",
      "php-class-tree"
    ],
    "packagesWithKeymapsDisabled": [
      "regex-railroad-diagram"
    ],
    "themes": [
      "atom-dark-ui",
      "one-dark-syntax"
    ]
  }
}
```

### Installed Packages

```coffee
# User
aligner, v0.18.1
aligner-php, v1.0.0
aligner-python, v1.0.1
aligner-scss, v1.1.0
atom-html-preview, v0.1.13
autoclose-html, v0.20.0
autocomplete-paths, v1.0.2
autocomplete-php, v0.3.7
autocomplete-python, v1.5.2
autocomplete-xml, v0.8.1
autohide-tree-view, v0.25.1
color-picker, v2.1.1
docblock-format-helper, v0.3.0
docs-snippets, v0.8.0
editor-stats, v0.17.0
emmet, v2.4.1
figlet, v0.5.3
file-icons, v1.6.14
foldername-tabs, v0.1.10
highlight-column, v0.5.0
highlight-selected, v0.11.2
keyboard-localization, v1.4.14
language-apache, v1.4.0
language-autoit, v0.2.1
language-batch, v0.3.3
language-ini, v1.14.0
language-nginx, v0.6.1
language-pascal, v0.17.0
language-patch, v0.1.0
line-count-status, v0.0.3
linter, v1.11.3
linter-alex, v1.7.0
linter-eslint, v6.0.0
linter-htmlhint, v1.0.2
linter-pep8, v1.2.1
linter-php, v1.1.7
linter-phpmd, v1.4.1
linter-pylama, v0.2.6
linter-tidy, v2.1.0
markdown-preview-opener, v0.1.1
markdown-scroll-sync, v2.0.3
minimap, v4.19.0
minimap-bookmarks, v0.2.0
minimap-codeglance, v0.4.5
minimap-cursorline, v0.1.0
minimap-find-and-replace, v4.4.0
minimap-git-diff, v4.1.8
minimap-highlight-selected, v4.3.1
minimap-linter, v1.1.1
minimap-pigments, v0.1.7
minimap-selection, v4.3.1
minimap-split-diff, v0.3.0
opened-files, v0.2.8
package-cop, v0.2.10
php-cs-fixer, v2.5.2
php-debug, v0.2.1
pigments, v0.23.0
pretty-json, v0.4.1
python-isort, v0.0.7
python-tools, v0.6.7
python-yapf, v0.9.1
regex-railroad-diagram, v0.11.0
sort-lines, v0.14.0
switcheroo, v1.1.0
tablr, v0.10.0
tabs-to-spaces, v1.0.1
tidy-markdown, v1.0.0

# Dev
No dev packages
```

**Comment by rugk** [comment]
Previous issue is obviously this: https://github.com/atom/find-and-replace/issues/619

**Comment by 50Wliu** [comment]
This has already been fixed and at the current pace of development will be available in Atom 1.6.0.

**Comment by rugk** [comment]
Okay, thanks for the information.
